### DISCUSSION TITLE
Welcome to linkfree_clone Discussions!

### DISCUSSION THREAD

**Discussion by himanshu1221** [himanshu1221/linkfree_clone#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RealHenryMartinez** [comment]
# [IDEAS] - Form Page and/or Edit cards

Would a form page to upload data to a sample JSON file in the source folder to localstorage be needed here for a way to look at simple "backend" functionality all through frontend? 
Or would an edit items on each card work better in this case?

If so, could I work on i?

**Reply by himanshu1221** [reply]
Go ahead and give it a try

**Reply by RealHenryMartinez** [reply]
Sounds great, thanks
